sechub-sereco/build.gradle

```diff
@@ -5,11 +5,11 @@
  * Root build file: "${rootProject.projectDir}/build.gradle"
  * ============================================================================
  */
+ 
 dependencies {
 
+	compile 'com.contrastsecurity:java-sarif:1.0'
```

Comment:
Please move the dependency name into `libraries.gradle` and use the key/name to reference it in compile.

Reply:
done